webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js

```diff
@@ -131,6 +140,133 @@ class SakaiRubricsManager extends RubricsElement {
+          $('.site_name.sort_element_site').removeClass('collpase-icon fa fa-chevron-down');
+          $('.site_name.sort_element_site').addClass('collpase-icon fa fa-chevron-up');
+          ascending = true;
+        }
+        break;
+      case 'site_title':
+        if ($('.site_title.sort_element_site').hasClass('collpase-icon fa fa-chevron-up')) {
+          $('.site_title.sort_element_site').removeClass('collpase-icon fa fa-chevron-up');
+          $('.site_title.sort_element_site').addClass('collpase-icon fa fa-chevron-down');
+          ascending = false;
+        } else {
+          $('.site_title.sort_element_site').removeClass('collpase-icon fa fa-chevron-down');
+          $('.site_title.sort_element_site').addClass('collpase-icon fa fa-chevron-up');
+          ascending = true;
+        }
+        break;
+      case 'creator_name':
+        if ($('.creator_name.sort_element_site').hasClass('collpase-icon fa fa-chevron-up')) {
+          $('.creator_name.sort_element_site').removeClass('collpase-icon fa fa-chevron-up');
+          $('.creator_name.sort_element_site').addClass('collpase-icon fa fa-chevron-down');
+          ascending = false;
+        } else {
+          $('.creator_name.sort_element_site').removeClass('collpase-icon fa fa-chevron-down');
+          $('.creator_name.sort_element_site').addClass('collpase-icon fa fa-chevron-up');
+          ascending = true;
+        }
+        break;
+      case 'modified':
+        if ($('.modified.sort_element_site').hasClass('collpase-icon fa fa-chevron-up')) {
+          $('.modified.sort_element_site').removeClass('collpase-icon fa fa-chevron-up');
+          $('.modified.sort_element_site').addClass('collpase-icon fa fa-chevron-down');
+          ascending = false;
+        } else {
+          $('.modified.sort_element_site').removeClass('collpase-icon fa fa-chevron-down');
+          $('.modified.sort_element_site').addClass('collpase-icon fa fa-chevron-up');
+          ascending = true;
+        }
+        break;
+    } 
+
+    let elementChildSite= this.querySelector("sakai-rubrics-list");
+    elementChildSite.sortRubrics(sortInputValue, ascending);
+  }
+
+  sortSharedRubrics(event) {
+    var sortInput = event.target.className;
+    var sortInputValue = sortInput.toString().toLowerCase();
+    var ascending = true; 
+    //icon from header
+    $('.sort_element_shared').each( function( i, val ) {
+      var className = $( val ).attr('class');
+      if (!className.includes(sortInputValue)) {
+        $( val ).removeClass('collpase-icon fa fa-chevron-down');
+        $( val ).removeClass('collpase-icon fa fa-chevron-up');
+      }
+    });
+    switch (sortInputValue) {
+      case 'site_name':
+        if ($('.site_name.sort_element_shared').hasClass('collpase-icon fa fa-chevron-up')) {
+          $('.site_name.sort_element_shared').removeClass('collpase-icon fa fa-chevron-up');
+          $('.site_name.sort_element_shared').addClass('collpase-icon fa fa-chevron-down');
```

Comment:
Why are you removing two classes and then adding them again? This should all be vanilla js, too. No need for jQuery in any of this.